resources/js/processes-catalogue/components/ProcessesCatalogue.vue

```diff
@@ -136,9 +139,28 @@ export default {
     }
   },
   methods: {
+    getUserConfiguration() {
+      ProcessMaker.apiClient
+        .get(this.urlConfiguration)
+        .then((response) => {
+          this.userConfiguration = JSON.parse(response.data.ui_configuration);
+          this.showMenu = this.userConfiguration.launchpad.isMenuCollapse;
+        });
```

Comment:
is there a possibility to get the entire configuration from the Controller?
So we avoid making several Call to API